metadata_layer.py

```diff
@@ -943,7 +943,7 @@ def from_edition(cls, edition):
         for contribution in edition.contributions:
             contributor = ContributorData.from_contribution(contribution)
             contributors.append(contributor)
-        else:
+        if not edition.contributions:
```

Comment:
Is there ever a case when a duplicate contribution makes sense, like maybe when an author is also the translator?